### PR TITLE
added dropdown for item 2

### PR DESCRIPTION
**Pull request by FahadBinQaiser**
Added JS for dropdown 2

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -160,7 +160,7 @@ <h1 class="text-sm leading-[1] text-[#555] font-bold">
               <p class="text-gray-500 text-[13px]">Add dates</p>
             </div>
             <div
-              class="rounded text-left p-2 z-[2] shadow-2xl bg-white absolute w-[810px] left-[-250px] top-[70px] rounded-[30px] h-[410px] hidden"
+              class="rounded text-left p-2 z-[2] shadow-2xl bg-white absolute w-[830px] left-[-270px] top-[70px] rounded-[30px] h-[450px] hidden"
```
Comment (farazapoex) [inline index.html:163]: Avoid writing tailwind classes like this ` w-[830px]` use the standard classes if possible

File: index.html [hunk 2]
```diff
@@ -212,15 +215,42 @@ <h2 class="text-[17px] font-medium text-center mb-4">
...
                 </div>
               </div>
 
               <div id="months-section" class="section hidden">
-                <h2 class="text-center text-lg">Months Content</h2>
+                <h2 class="text-center text-lg">
+                  Faraz Bhai, I think the round month bar in Airbnb website
+                  can't be made through the tech stack we are using ( html,
+                  tailwind css, Js ) and although I thought to ask you for this
+                  thing. Let me know about this.
```
Comment (farazapoex) [inline index.html:232]: Sure it can.

File: script.js [hunk 3]
```diff
@@ -313,12 +313,12 @@ function generateCalendar(days, elementId, startDay, disableUntil = 0) {
   for (let i = 1; i <= days; i++) {
     const dayElement = document.createElement('div');
     dayElement.textContent = i;
-    dayElement.classList.add('text-center', 'p-2', 'rounded');
+    dayElement.classList.add('text-center','text-sm', 'p-2', 'w-10', 'h-10', 'rounded', 'border-2', 'border-transparent');
```
Comment (farazapoex) [inline script.js:316]: use an array instead

File: script.js [hunk 4]
```diff
@@ -337,4 +337,140 @@ function showSection(clickedTab) {
...
+  const container = document.getElementById("container10");
+  container.classList.add("flex", "flex-row", "gap-2", "overflow-hidden", "mt-2");
+
+  const visibleMonths = months.slice(0, 6);
+  visibleMonths.forEach((month) => {
+
+    const monthBox = document.createElement("div");
+
+    // Add classes for each box
+    monthBox.classList.add(
```
Comment (farazapoex) [inline script.js:416]: Array and iterate through it

**Comment by farazapoex** [comment]
Please stop using AI

File: index.html (hunk 1 above)
Comment (FahadBinQaiser) [reply index.html:163]: okay
But for larger units like 830px I don't have a value that perfectly suits the size.
I tried from the tailwind CSS docs and they have properties but less than this,

File: index.html (hunk 2 above)
Comment (FahadBinQaiser) [reply index.html:232]: it can be made with the tech stack we are using????
Clear me this one please.

File: script.js (hunk 3 above)
Comment (FahadBinQaiser) [reply script.js:316]: okay

File: script.js (hunk 4 above)
Comment (FahadBinQaiser) [reply script.js:416]: i didn't understand this one!

**Comment by FahadBinQaiser** [comment]
> Please stop using AI

Faraz bhai I have to say that I try everytime but when I don't get the thing then I visit AI for some information and help.
I don't get information from other sources In the end so I make my way to AI

File: script.js (hunk 3 above)
Comment (FahadBinQaiser) [reply script.js:316]: it's done,
I used an array.

File: script.js (hunk 4 above)
Comment (FahadBinQaiser) [reply script.js:416]: its done

Comment (farazapoex) [reply script.js:416]:
```
const classes = [
  "flex",
  "flex-col",
  "items-center",
  "justify-center",
  "border-2",
  "p-2",
  "rounded-[30px]",
  "w-[130px]",
  "h-36",
  "cursor-pointer",
  "hover:border-black"
];

classes.forEach(className => monthBox.classList.add(className));
```

Comment (FahadBinQaiser) [reply script.js:416]: it's done

**Pull request merged**